examples/derive_ref/README.md

```diff
@@ -114,7 +114,13 @@ translated into a mere method call.
 These correspond to a `clap::Command` which is used for both top-level parsers and
 when defining subcommands.
 
-In addition to the raw attributes, the following magic attributes are supported:
+#### Raw attributes
+
+- command attributes are forwarded to all setters on [clap::App](https://docs.rs/clap/latest/clap/struct.App.html#application-wide-settings)
+  - e.g. `arg_required_else_help = true` would perform `.arg_required_else_help(true)` on [App::arg_required_else_help](https://docs.rs/clap/latest/clap/struct.App.html#method.arg_required_else_help)
```

Comment:
The document is currently organized as
- Raw Attributes
- Magic Attributes
  - Command Attributes
  - Arg Attributes
  - ...

This change is switching it to 

- Raw Attributes
- Magic Attributes
  - Command Attributes
    - Raw attributes
    - Magic attributes
  - Arg Attributes
    - Raw attributes
    - Magic attributes
  - ...

which is redundant.

Reply:
good point.

I reorganized to:
- Raw Attributes
-  Magic Attributes
-  List of attributes
   -   Command Attributes
       -   Raw attributes
       -   Magic attributes
    - Arg Attributes
         -   Raw attributes
         -   Magic attributes
      
Listing all of the attributes together makes the most sense to me.
But if you need it done some other way please let me know, at the end of the day the most important thing is to link to where these raw attributes come from.

Reply:
I've done some tweaking of the ideas here in #3529 and would be interested in your thoughts.

Reply:
Nice!
That is clearer than what I had.
I'll go ahead and close this PR.